cmd/tailscale/cli/web.go

```diff
@@ -60,6 +60,15 @@ type tmplData struct {
 	LicensesURL       string
 	TUNMode           bool
 	IsSynology        bool
+	DSMVersion        int
```

Comment:
what values can this have? 6, 7, or 0 to mean not synology? doc? Oh, we have IsSynology. So say "// 6 or 7, if IsSynology"